### ISSUE TITLE
Update radio button[name="amount"] group semantic structure/aria labels

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#318]
### Issue: [Radio inputs with the same name attribute value must be part of a group](https://dequeuniversity.com/rules/axe/3.2/radiogroup?application=AxeChrome)

**Element path:** `#one-time-amount-0`, `#one-time-amount-1`, `#one-time-amount-2`, `#one-time-amount-3`, `#one-time-amount-other`, `#monthly-amount-0`, `#monthly-amount-1`, `#monthly-amount-2`, `#monthly-amount-3`, `#monthly-amount-other`

**Snippet(s):** _multiple instances, see above_

**How to fix:**
Fix any of the following:
- Elements with the name "amount" do not all have both a shared label, and a unique label, referenced through aria-labelledby
- Element does not have a containing fieldset or ARIA group

====

More Info:

Comments: n/a
Resources: n/a

**Comment by nicklee** [comment]
I've added role="radiogroup" to .donate-form__fields (the container around all of the .donation-amount divs that container the label and radio field), updating the elements to have an aria-labelledby is a little more involved (the forms are populated via javascript) so will add this to the backlog as a nice to have if that's OK?

**Comment by nicklee** [comment]
https://github.com/mozilla/donate-wagtail/pull/364

**Comment by youriwims** [comment]
Yep, this sounds good!

**Comment by Pomax** [comment]
closing as out of date,,,,,,
